docs/docs/getting-started/logging-first-model/index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+sidebar-position: 3
+---
+
+# Tutorial Overview
+
+In this entry point tutorial to MLflow, we'll be covering the essential basics of core MLflow functionality associated
+with tracking training event data.
+
+We'll start by learning how to start a local MLflow Tracking server, how to access and view the MLflow UI, and move on to
+our first interactions with the Tracking server through the use of the MLflow Client.
+
+The tutorial content builds upon itself, culminating in successfully logging your first MLflow model.
+
+The topics in this tutorial cover:
+
+- Starting an **MLflow Tracking Server** (Optionally) and connecting to a Tracking Server
+- Exploring the **MlflowClient** API (briefly)
+- Understanding the Default Experiment
+- **Searching** for Experiments with the MLflow client API
+- Understanding the uses of **tags** and how to leverage them for model organization
+- Creating an Experiment that will contain our **run** (and our model)
+- Learning how to **log** metrics, parameters, and a model artifact to a run
+- Viewing our Experiment and our first run within the **MLflow UI**
+
+To get started with the tutorial, click NEXT below or navigate to the section that you're interested in:
+
+- [Starting an MLflow Tracking Server](step1-tracking-server)
+- [Using the MLflow Client API](step2-mlflow-client)
+- [Creating Experiments](step3-create-experiment)
+- [Searching Experiments](step4-experiment-search)
+- [Create a dataset about apples](step5-synthetic-data)
+- [Logging our first runs with MLflow](step6-logging-a-run)
+- [Logging Your First MLflow Model Notebook](notebooks)
+
+If you would instead like to download a notebook-based version of this guide and follow along locally, you can download the notebook from the link below.
+
+<a class="button button--primary" download href="/notebooks/logging-first-model.ipynb">
```

Comment:
would it be possible to put the `.ipynb` together in the same folder (`logging-first-model/notebooks/logging-first-model.ipynb`)? The reason for this is the directory structure needs to be preserved for the future `.ipynb -> .mdx` conversion process.

i understand that downloads don't work unless the file is in the static folder, but maybe we can preserve the behaviour of downloading the notebook from GitHub. Could we create a custom react component like:

```jsx
<NotebookDownloadButton href="https://raw.githubusercontent.com/...">
  Title
</NotebookDownloadButton>
```

The existing implementation creates a blob from the provided href, and then dumps the content into an invisible button and simulates a click to download. This seemed unnecessarily complicated to me, but after googling it seems like there isn't really a better solution 😵‍💫:

https://github.com/mlflow/mlflow/blob/3eb016f16806cac5c52dbcc0eb6206fd2705a41d/docs/theme/mlflow/layout.html#L230-L251

Reply:
Yes, download files from different domains isn't possible unless we can configure the response headers returned by the server. I'll implement this `NotebookDownloadButton` React component you are suggesting.